src/wui/wui.c

```diff
@@ -47,9 +48,7 @@ void update_web_vars(void) {
     web_vars.print_dur = wui.wui_marlin_vars->print_duration;
     web_vars.sd_precent_done = wui.wui_marlin_vars->sd_percent_done;
     web_vars.sd_printing = wui.wui_marlin_vars->sd_printing;
-    if (marlin_event(MARLIN_EVT_GFileChange)) {
-        marlin_get_printing_gcode_name(web_vars.gcode_name);
-    }
+    web_vars.gcode_name = wui.wui_marlin_vars->media_file_name;
```

Comment:
how the other clients get informed about the file change?

Reply:
filename is now variable, client can check change flag for this variable:
if (marlin_change_clr(MARLIN_VAR_FILENAME))
filename variable is changed when printing started